src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -300,7 +311,7 @@ export class EstimatedQueryPlanAction extends QueryTaskbarAction {
 		@IConnectionManagementService connectionManagementService: IConnectionManagementService
 	) {
 		super(connectionManagementService, editor, EstimatedQueryPlanAction.ID, EstimatedQueryPlanAction.EnabledClass);
-		this.label = nls.localize('estimatedQueryPlan', "Explain");
+		this.label = nls.localize('estimatedQueryPlan', "Estimated Plan");
```

Comment:
Show Estimated Plan?

Reply:
Won't the text be too long for the toolbar?

Reply:
Yeah, I think we should hold off on string changes unless we have direct feedback from the PMs about one way or another and then plan to follow up with doing a focus session or something later if needed.